lib/base/Domain.js

```diff
@@ -21,7 +20,9 @@ function Domain(twilio, baseUrl) {
  * @return {string} absolute url
  */
 Domain.prototype.absoluteUrl = function(uri) {
-  return _.trim(this.baseUrl, '/') + '/' + _.trim(uri, '/');
+  const cleanUri = uri.replaceAll('/', '');
+  const cleanBaseUrl = this.baseUrl.replaceAll('/','');
+  return cleanBaseUrl + '/' + cleanUri;
```

Comment:
`/` within the uri/baseUrl will be overwritten. Lodash trim only removes the leading and trailing characters which its not the same as `replaceAll`.

A potential workaround is to maybe create our own trim function?

```
const trim = (str, c = '\\s') =>
 str.replace(new RegExp(`^([${c}]*)(.*?)([${c}]*)$`), '$2')
```

```
console.log(trim("/A/BC/DCF/","/"))
=> A/BC/DCF 
```